onboarding/src/routes/coordinator.ts

```diff
@@ -18,3 +18,7 @@ export const nextPage = (history: History, offset: number): void => {
 export const previousPage = (history: History, offset: number): void => {
   history.push(`/user-list/${offset - 10}`);
 };
+
+export const goToUserPage = (history: History, idparams: string): void => {
```

Comment:
Na taq usamos snakeCase pra variáveis
```suggestion
export const goToUserPage = (history: History, idParams: string): void => {
```